Bottom: 2bf20766ef8212c371af1f079c671b51552d1b3e
Top:    5b77e2690426d9c4f990dc4fd02472eb0cac096b
Author: Lawrence D'Anna <lawrence_danna@apple.com>
Date:   2019-10-04 11:18:20 -0700

Refresh of remove-a-smattering-of-0

---
diff --git a/lldb/source/Core/IOHandler.cpp b/lldb/source/Core/IOHandler.cpp
index 5b9c760ad21..b134b69456e 100644
--- a/lldb/source/Core/IOHandler.cpp
+++ b/lldb/source/Core/IOHandler.cpp
@@ -329,7 +329,7 @@ bool IOHandlerEditline::GetLine(std::string &line, bool &interrupted) {
 
         if (prompt && prompt[0]) {
           if (m_output_sp) {
-            m_output_sp->GetFile().Printf("%s", prompt);
+            m_output_sp->Printf("%s", prompt);
             m_output_sp->Flush();
           }
         }
@@ -490,9 +490,9 @@ bool IOHandlerEditline::GetLines(StringList &lines, bool &interrupted) {
       std::string line;
       if (m_base_line_number > 0 && GetIsInteractive()) {
         if (m_output_sp) {
-          m_output_sp->GetFile().Printf(
-              "%u%s", m_base_line_number + (uint32_t)lines.GetSize(),
-              GetPrompt() == nullptr ? " " : "");
+          m_output_sp->Printf("%u%s",
+                              m_base_line_number + (uint32_t)lines.GetSize(),
+                              GetPrompt() == nullptr ? " " : "");
         }
       }
 
diff --git a/lldb/source/Expression/REPL.cpp b/lldb/source/Expression/REPL.cpp
index 134c16fedee..4f81ee3e56d 100644
--- a/lldb/source/Expression/REPL.cpp
+++ b/lldb/source/Expression/REPL.cpp
@@ -423,7 +423,7 @@ void REPL::IOHandlerInputComplete(IOHandler &io_handler, std::string &code) {
               .SetBaseLineNumber(m_code.GetSize() + 1);
         }
         if (extra_line) {
-          output_sp->GetFile().Printf("\n");
+          output_sp->Printf("\n");
         }
       }
     }
